### PR TITLE
feat: add new default metrics

### LINKED ISSUE #228: Add an rmsle evaluation metric?
## Feature request

The implementation of a root [mean squared logarithmic error regression loss](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_squared_log_error.html) (`rmsle`) evaluation metric as an out-of-the-box option in TabNet, as it is frequently used in kaggle competitions.
I know one can write a little python class, to do this, but having it as an option would be even easier!

**What is the expected behavior?**

return the rmsle

**What is motivation or use case for adding/changing the behavior?**

This particular metric is often used in kaggle competitions.

**How should this be implemented in your opinion?**

A simple addition+modification based on the code section used to calculate/evaluate the `rmse` 

**Are you willing to work on this yourself?**
Sorry, no, I would probably ruin the whole code...

### PR DESCRIPTION
**Pull request by Optimox**
**What kind of change does this PR introduce?**

Adds a few metrics, update README and fix a minor f-string bug.

**Does this PR introduce a breaking change?**
Nope

**What needs to be documented once your changes are merged?**
Already done in the README

**Closing issues**
Closes #228

### CODE REVIEW

**Review by eduardocarvp — APPROVED** [review]

**Comment by eduardocarvp** [comment]
The CI error was a timeout. I `rerun from failed`, let's see.

File: pytorch_tabnet/metrics.py [hunk 1]
```diff
@@ -347,6 +348,39 @@ def __call__(self, y_true, y_score):
...
+            Target matrix or vector
+        y_score : np.ndarray
+            Score matrix or vector
+
+        Returns
+        -------
+        float
+            RMSLE of predictions vs targets.
+        """
+        y_score = np.clip(y_score, a_min=0, a_max=None)
```
Comment (eduardocarvp) [inline pytorch_tabnet/metrics.py:380]: Shouldn't `a_min=eps` in this case?
